src/server/utils/onModuleLoad.js

```diff
@@ -118,7 +118,9 @@ export default function onModuleLoad({
   }
 
   if (providedExternals) {
-    throw new Error(`Module ${moduleName} attempted to provide externals, but it is not the root module.`);
+    console.warn(
+      `Module ${moduleName} attempted to provide externals. Only the root module can provide externals.`
+    );
```

Comment:
You will need some changes around here: https://github.com/americanexpress/one-app-cli/blob/master/packages/one-app-bundler/webpack/loaders/provided-externals-loader.js#L34

Reply:
I think it's best we move that helper added by the loader into One App -> `getRootModule` and remove it from being injected. Thoughts?

Reply:
- [ ] Get @10xLaCroixDrinker Feedback on this